db/migrations/0003_alter_actor_first_name_alter_actor_last_name_and_more.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 4.0.2 on 2024-03-17 15:54
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_actor_first_name_actor_last_name_genre_name'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='actor',
+            name='first_name',
+            field=models.CharField(max_length=255),
+        ),
+        migrations.AlterField(
+            model_name='actor',
+            name='last_name',
+            field=models.CharField(max_length=255),
+        ),
+        migrations.AlterField(
+            model_name='genre',
+            name='name',
+            field=models.CharField(max_length=255),
+        ),
+    ]

```

Comment:
It is a good practice to include one migration per app per PR in Django projects since it helps to avoid `migrations` folder overload in the future. Represent all changes done to models in one migration file per app per PR, please